### ISSUE TITLE
Additional server-side (PHP) checking libraries

### ISSUE THREAD

**Issue by nightrider77** [igorescobar/jQuery-Mask-Plugin#794]
Hello,

Are there any PHP libraries that can understand masks of this plugin and additionally validate user input on the back end of the application?

**Comment by douglas-legulas** [comment]
I don't think so. We are instructed to "unmask" the value before doing anything with it. Then you would use it as a float/decimal or integer.
